repairnator/src/main/java/fr/inria/spirals/repairnator/serializer/gsheet/GoogleSpreadSheetFactory.java

```diff
@@ -33,8 +33,9 @@
     private Logger logger = LoggerFactory.getLogger(GoogleSpreadSheetFactory.class);
     private static final String APPLICATION_NAME = "RepairNator Bot";
     private static final File DATA_STORE_DIR = new File(System.getProperty("user.home"), ".credentials/sheets.googleapis.com-repairnator");
-    private static final String GOOGLE_SECRET_PATH = "./client_secret.json";
-    public static final String SPREADSHEET_ID = "1FUHOVx1Y3QZCAQpwcrnMzbpmMoWTUdNg0KBM3NVL_zA";
+    private static final String GOOGLE_SECRET_PATH = "/home/fernanda/git/librepair/repairnator/client_secret.json";
+    //public static final String SPREADSHEET_ID = "1FUHOVx1Y3QZCAQpwcrnMzbpmMoWTUdNg0KBM3NVL_zA";
+    public static final String SPREADSHEET_ID = "1MnRwoZGCxxbmkiswc0O6Rg43wJFTBc3bIyrNdTiBhQ4";
 
```

Comment:
I don't follow your thought. First, it seems you want to do different things with ```GOOGLE_SECRET_PATH``` and ```SPREADSHEET_ID```, and then it seems you want to use the same solution for both. Another ideia, but I'm not sure if it's good: can we do something like a ```Constants``` class e depending on the ```mode```, these constants assume different values?